src/windows/enumerate.rs

```diff
@@ -303,9 +306,7 @@ impl PortDevice {
 
         let port_name = &port_name_buffer[0..port_name_len as usize];
```

Comment:
Don't we have to check account for the different buffer types `u8` versus `u16` as for `RegEnumValueW` below?

Reply:
fixed